### DISCUSSION TITLE
Rust analyzer for the web

### DISCUSSION THREAD

**Discussion by Ben2146053** [rust-lang/rust-analyzer#16410] [General]
I have a use case for bringing rust analyzer features to the web, exposing it as a web API. The idea is sending  code to be processed and getting a response of the rust analyzer output. Is there anything out in the wild already doing this? If not, can anyone point me in the repo where a good starting point to look into? Thank you.

**Comment by hkalbasi** [comment]
This might be interesting: https://github.com/qualified/lsps/tree/main/examples/rust-analyzer
